tests/uber/models/test_attendee.py

```diff
@@ -253,8 +254,8 @@ def test_self_service_refunds_if_on(monkeypatch, open, expected):
     monkeypatch.setattr(config.Config, 'SELF_SERVICE_REFUNDS_OPEN',
                         property(open))
     attendee = Attendee(paid=c.HAS_PAID, amount_paid=10)
-    monkeypatch.setattr(Attendee, 'stripe_transactions',
-                        [StripeTransaction(fk_id=attendee.id, amount=1000)])
+    txn = StripeTransaction(amount=1000)
+    txn.attendees = [StripeTransactionAttendee(attendee_id=attendee.id, txn_id=txn.id, share=1000)]
```

Comment:
I attempted to figure out how to update the unit tests, but didn't quite manage to make this one pass again.